src/sonic_ax_impl/mibs/vendor/cisco/ciscoEntityFruControlMIB.py

```diff
@@ -18,38 +52,57 @@ def __init__(self):
         """
         init the handler
         """
-        self.psuutil = None
+        self.statedb = SonicV2Connector()
+        self.statedb.connect(self.statedb.STATE_DB)
 
-        try:
-            module = imp.load_source(PSU_PLUGIN_MODULE_NAME, PSU_PLUGIN_MODULE_PATH)
```

Comment:
>load_source [](start = 25, length = 11)

Since we don't need the module 'psutil', can we remove it from setup.py?

Reply:
@qiluo-msft Unfortunately we can't remove it from setup.py

**Summary:**
From [psutil](https://pypi.org/project/psutil/):

psutil (process and system utilities) is a cross-platform library for retrieving information on running processes and system utilization (CPU, memory, disks, network, sensors) in Python.

**Usage:**
From [sonic_ax_impl](https://github.com/Azure/sonic-snmpagent/blob/master/src/sonic_ax_impl/mibs/vendor/__init__.py):

```
# From the psutil documentation https://pythonhosted.org/psutil/#psutil.cpu_percent:
#
#    Warning the first time this function is called
#    with interval = 0.0 or None it will return a
#    meaningless 0.0 value which you are supposed
#    to ignore.
psutil.cpu_percent()
# '...is recommended for accuracy that this function be called with at least 0.1 seconds between calls.'
time.sleep(0.1)
# a sliding window of 60 contiguous 5 sec utilization (up to five minutes)
self.cpuutils = collections.deque([psutil.cpu_percent()], maxlen=60)
self.system_virtual_memory = psutil.virtual_memory()
```